src/components/ConnectSafe/index.tsx

```
@@ -129,26 +129,26 @@ export default function ConnectSafe() {
         safeAddress: getAddress(safeFromUrl),
         moduleAddress: getAddress(moduleFromUrl)
       });
     }
     else if (safes.length > 0 && !safeAddress && signer && ownerAddress) {
-      try{
+      try {
         //@ts-ignore
-        let localStorage: {[key: string]:{safeAddress: string, moduleAddress: string}} = loadStateFromLocalStorage(`staking-hub-chosen-safe`);
-        if(localStorage && localStorage[ownerAddress] && safes.filter(safe=>safe?.safeAddress === localStorage[ownerAddress]?.safeAddress).length > 0){
+        let localStorage: { [key: string]: { safeAddress: string, moduleAddress: string } } = loadStateFromLocalStorage(`staking-hub-chosen-safe`);
+        if (localStorage && localStorage[ownerAddress] && safes.filter(safe => safe?.safeAddress === localStorage[ownerAddress]?.safeAddress).length > 0) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)